evalai/challenges.py

```diff
@@ -11,66 +11,43 @@
 
 @click.group(invoke_without_command=True)
 @click.pass_context
-def challenges(ctx):
-    """
-    Challenges and related options.
-    """
-    if ctx.invoked_subcommand is None:
-        welcome_text = ("Welcome to the EvalAI CLI. Use evalai"
-                        "challenges --help for viewing all the options.")
-        echo(welcome_text)
-
-
-@click.group(invoke_without_command=True, name='list')
-@click.pass_context
 @click.option('--participant', is_flag=True,
               help="Show the challenges that you've participated")
 @click.option('--host', is_flag=True,
               help="Show the challenges that you've hosted")
-def list_challenges(ctx, participant, host):
+def challenges(ctx, participant, host):
     """
     Used to list challenges.
-    Invoked by running `evalai challenges list`
+    Invoked by running `evalai challenges`
     """
-    if participant:
-        get_challenge_count(host, participant)
-    elif host:
+    if participant or host:
```

Comment:
So here's a potential bug. The function being called when either participant or host is True first checks for host and if host is false, checks for participant.
Now if I activate both flags, I'll only ever get host challenges. You'll have to redesign the `get_challenge_count` function to account for this (and possibly rename it too).

Reply:
@varunagrawal I've added an issue for this. I'll resolve it.